File: OLD/Android/CheckoutPagamento/app/src/main/java/com/example/checkoutpagamento/MainActivity.java
package com.example.checkoutpagamento;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {


    private EditText edtValorCompra;
    private EditText edtValorPago;
    private TextView txtValorTroco;
    private TextView txtValorBrinde;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        edtValorCompra = findViewById(R.id.edtValorTotal);
        edtValorPago = findViewById(R.id.edtValorPago);

        txtValorTroco = findViewById(R.id.valueTroco);
        txtValorBrinde = findViewById(R.id.valueBrinde);
    }

    public void finalizarCompra (View view){

        double valorCompra = Double.parseDouble(edtValorCompra.getText().toString());
        double valorPago = Double.parseDouble(edtValorPago.getText().toString());

        double valorTroco = valorPago - valorCompra;

        txtValorTroco.setText("R$:" + valorTroco);

        if(valorCompra >= 100){
            txtValorBrinde.setText("SIM");

         }else {
            txtValorBrinde.setText("NAO");

        }



    }

}

File: OLD/Kotlin/buttonAnimation/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import android.util.Log
import android.view.Menu
import android.view.MenuItem
import android.view.animation.AnimationUtils
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        fun onCreateOptionsMenu(menu: Menu?): Boolean {
            menuInflater.inflate(R.menu.menu, menu)
            return true
        }

        fun onOptionsItemSelected(item: MenuItem?): Boolean {
            when (item?.itemId) {
                R.id.sobre -> {
                    Log.i("Teste", "Sobre clicado")
                }
            }
            return super.onOptionsItemSelected(item)
        }


        val lists = arrayOf("Sim", "Talvez", "Não conte com isso!", "teste")
        btn.setOnClickListener {
            val random = Random()
            val index = random.nextInt(4)
            txt_response.text = lists[index]
            Log.i("Teste", "O index é ${index}")
        }

        btn.setOnClickListener {
            val animation = AnimationUtils.loadAnimation(this, R.anim.button_anim)
            btn.startAnimation(animation)
            val random = Random()
            val index = random.nextInt(3)
            Log.i("Teste", "O index é ${index}")
            txt_response.text = lists[index]
        }





    }
}